web_src/js/features/comp/ConfirmModal.js

```diff
@@ -1,22 +1,23 @@
 import $ from 'jquery';
 import {svg} from '../../svg.js';
 import {htmlEscape} from 'escape-goat';
+import {createElementFromHTML} from '../../utils/dom.js';
 
 const {i18n} = window.config;
 
-export async function confirmModal(opts = {content: '', buttonColor: 'primary'}) {
+export function confirmModal({content = '', buttonColor = 'red'}) {
```

Comment:
Why default `buttonColor` is changed to `red`?

Reply:
Because I guess most confirm dialogs are because of dangerous actions.

Reply:
I guess no ..... I have used many different frameworks, none of the default "confirm" dialog buttons is a "negative" button.

Reply:
Changed it back to primary.